can_tools/scrapers/official/WY/WYVaccines.py

```diff
@@ -297,3 +297,184 @@ def normalize(self, data) -> pd.DataFrame:
+
+    state_fips = int(us.states.lookup("Wyoming").fips)
+    execution_dt = pd.Timestamp.now()
+    source = (
+        "https://health.wyo.gov/publichealth/immunization/wyoming-covid-19-vaccine-information/"
+        "covid-19-vaccine-distribution-data/"
+    )
+    source_name = "Wyoming Department of Health"
+    baseUrl = "https://datastudio.google.com/batchedDataV2"
+    resource_id = "a51cf808-9bf3-44a0-bd26-4337aa9f8700"
+    has_location = False
+    location_type = "county"
+
+    def fetch(self):
+        """
+        Pulls Wyoming county vaccine data, cleans and up the Json Strig that is returned before returning that object as
+        a dictionary
+        """
+        WYVacDataTotal = self.get_dataset(
+            json.loads(self.bodyTotalPercentage), url=self.baseUrl
+        )
+        parsedVacDataTotal = json.loads(WYVacDataTotal[10 : len(WYVacDataTotal) - 1])
+        WYVacData18P = self.get_dataset(
+            json.loads(self.body18UpPercentage), url=self.baseUrl
+        )
+        parsedVacData18P = json.loads(WYVacData18P[10 : len(WYVacData18P) - 1])
+        WYVacData65P = self.get_dataset(
+            json.loads(self.body65UpPercentage), url=self.baseUrl
+        )
+        parsedVacData65P = json.loads(WYVacData65P[10 : len(WYVacData65P) - 1])
+        df = parsedVacDataTotal["default"]["dataResponse"][0]["dataSubset"][0][
+            "dataset"
+        ]["tableDataset"]["column"]
+        df.append(
+            parsedVacData18P["default"]["dataResponse"][0]["dataSubset"][0]["dataset"][
```

Comment:
I didn't see a "default" key in the data, and removing it worked

```suggestion
            parsedVacData18P["dataResponse"][0]["dataSubset"][0]["dataset"][
```